apps/web/src/components/connection/connectionView.tsx

```diff
@@ -1,24 +1,20 @@
+"use client";
 import {
     Avatar,
     Button,
-    Collapse,
     Group,
-    ScrollArea,
+    Stack,
     Text,
     Title,
-    VisuallyHidden,
     useMantineTheme,
+    VisuallyHidden,
 } from "@mantine/core";
-import { useDisclosure, useViewportSize } from "@mantine/hooks";
-import { TbChevronDown, TbChevronUp, TbPlus } from "react-icons/tb";
+import { TbPlus } from "react-icons/tb";
 import { useConnectionConfig } from "../../providers/connectionConfig/hooks";
 import ConnectionInfo from "./connectionInfo";
 
 const ConnectionView = () => {
-    const [showConnectionList, { toggle: toggleConnectionList }] =
-        useDisclosure(true);
     const { showConnectionModal } = useConnectionConfig();
```

Comment:
I think the `showConnectionModal` and `listConnections` could be on the same line

```suggestion
    const { showConnectionModal, listConnections } = useConnectionConfig();
```